LocalMultiplayerAgent/DeploymentTool/deployment.json

```diff
@@ -0,0 +1,16 @@
+{
+    "BuildName": "GameContainerW",
+    "VmSize": "Standard_D2_v2",
+    "MultiplayerServerCountPerVm": 10,
+    "RegionConfigurations": [
+        {
+            "Region": "EastUs",
+            "MaxServers": 10,
+            "StandbyServers": 5
+        }
+    ],
+    "OSPlatform": "Windows",
```

Comment:
I think we can deduce what OSPlatform the customer wants without this field. If it's a managed container based build, we know it's Windows. If it's a custom container-based server, we know it's Linux. If it's a process-based server, LMA can assume it's Windows because LMA doesn't support Linux process-based servers yet.
Having this field here just enables developers to run their Windows process-based servers in LMA and then deploy them as Linux process-based servers (which will not work out for them)